cardscan-base/src/main/java/com/getbouncer/cardscan/base/ScanBaseActivity.java

```diff
@@ -504,7 +514,11 @@ public void onPreviewFrame(byte[] bytes, Camera camera) {
 
     @Override
     public void onClick(View view) {
-        if (mCamera != null && mFlashlightId == view.getId()) {
+        if (mEnterCardManuallyId == view.getId() && mEnterCardManuallyId != View.NO_ID) {
+            // this is essentially clicking the back button?
```

Comment:
it's not actually clicking the back button. it's closing this view and returning a specific result to the caller. the caller can then determine what to do with that result.

Reply:
whoops, old comment